src/main/java/lottery/domain/LotteryTicket.java

```diff
@@ -4,7 +4,8 @@
 
 public class LotteryTicket {
     public static final int NUMBER_OF_LOTTERY_NUMBERS = 6;
-    private final List<LotteryNumber> lotteryNumbers;
+//    private final List<LotteryNumber> lotteryNumbers;
+    private final SortedSet<LotteryNumber> lotteryNumbers;
```

Comment:
```suggestion
    private final Set<LotteryNumber> lotteryNumbers;
```
마찬가지로 구체적인 클래스로 선언하기 보단 상위 클래스로 선언하여 유연하게 구현해보세요. 
https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java

Reply:
![image](https://user-images.githubusercontent.com/8574015/101368465-87655580-38ea-11eb-990e-592f0fe45e6b.png)

해당 그림 통해서 확인했습니다. 유연하게 가려면 Set으로 해야 하는게 옮은 방향이었네요 👍